dashboard/api_test.go

```diff
@@ -22,16 +22,19 @@ const (
 func Test(t *testing.T) { TestingT(t) }
 
 type S struct {
-	srv *httptest.Server
+	srv        *httptest.Server
+	cookiePath string
```

Comment:
This looks unused?

Reply:
Oops, supposed to be used down in one the assertion for the Location header. I'll create another PR.

Reply:
https://github.com/flynn/flynn/pull/4403